src/radio_tool_button.rs

```diff
@@ -0,0 +1,23 @@
+use std::ptr;
+
+use RadioToolButton;
+use ToolItem;
+use ffi;
+use glib::object::Downcast;
+use glib::translate::*;
+
+impl RadioToolButton {
+    pub fn new() -> RadioToolButton {
+        assert_initialized_main_thread!();
+        unsafe {
+            ToolItem::from_glib_none(ffi::gtk_radio_tool_button_new(ptr::null_mut())).downcast_unchecked()
```

Comment:
Why `ToolItem` and not `RadioToolButton`? And generally, you can just use `from_glib_none` and let the compiler figure out the right implementation

Reply:
Oh good catch, didn't see it at all...

Reply:
It because says Gtk-3.0.gir:
```xml
        <return-value transfer-ownership="none">
          <doc xml:space="preserve">The new #GtkRadioToolButton</doc>
          <type name="ToolItem" c:type="GtkToolItem*"/>
        </return-value>

```

Reply:
I thought it would be best to be consistent with what gets auto-generated for similar functions, even if it does not really make a difference how one calls `from_glib_none`.